diffkemp/simpll/llvm-lib/14/FunctionComparator.h

```diff
@@ -320,6 +320,7 @@ class FunctionComparator {
   virtual int cmpTypes(Type *TyL, Type *TyR) const;
 
   virtual int cmpNumbers(uint64_t L, uint64_t R) const;
+  virtual int cmpAligns(Align L, Align R) const;
```

Comment:
I think I made a mistake when updating Diffkemp to LLVM 14 and added a wrong FunctionComparator file. I can move this into a separate PR if you wish, since it is unrelated to LLVM 15.

Reply:
No need, I was just curious why this happened.